README.md

```diff
@@ -13,5 +13,10 @@ A VS Code extension for Liberty dev mode. The extension will detect your Liberty
 - Run tests
 - View unit and integration test reports
 
+## Configurable User Settings
+| Setting | Description | Default Value |
+| --------  | ----------- | -------  |
+| liberty.terminal.useJavaHome | If this value is true, and if the setting `java.home` from the redhat extension has a value, then the environment variable `JAVA_HOME` will be set to the value of `java.home` when a new terminal window is created. | False |
```

Comment:
I don't think we need to say `from the redhat extension` since that variable also belongs in the same settings file, not a different place.  And `redhat extension` is ambiguous.